lib/menu.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'dish.rb'
+require_relative 'price_calculator.rb'
+
+class Menu
+  attr_reader :dishes, :price_list
+  def initialize(price_calculator_class: Price_calculator)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `price_calculator_class`. You can also write as `initialize(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)